src/components/Daos/DaosPage.tsx

```diff
@@ -67,11 +67,13 @@ class DaosPage extends React.Component<IProps, IState> {
     // If search string greater than 2 search on server for any other DAOs not yet loaded that match this search
     if (searchString.length > 2) {
       const arc = getArc();
+      const firstChar = searchString.charAt(0);
```

Comment:
Is it not feasible to make the entire DAO name search be case insensitive?

Reply:
I dont think so

Reply:
I mean it is if the DAOs are already loaded but for ones we look up on the server I didn't find how to do that with the graphnode

Reply:
OK I approved, but this looks kinda like a hack.  Maybe could add a code comment explaining the situation?

Reply:
i did below

Reply:
why not lower case the search string and and dao names and then do the search ?